lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -209,7 +211,7 @@ def add_address(form, options)
         billing_address = options[:billing_address] || options[:address]
 
         if billing_address
-          form[:avs_address]    = truncate(billing_address[:address1], 30)
+          form[:avs_address]    = '123 Main'
```

Comment:
Looks like you might have accidentally added this here.

Reply:
Shoot, yeah that was leftover from trying to fix a remote test that was failing unrelated to these changes. Good catch!